RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate E2EEnterYourPasswordView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce


- Click on the blue banner `Enter your E2E Password` in Rooms List
- The layout and functions should be working properly

## Screenshots

## Types of changes




## Checklist



## Further comments
